File: THOK/Fusion创建新模块所需的文件列表和文件/src/User Interface/Fusion/Extjs/Fusion/classic/src/Warehouse/model/Bill/ManualMaterialBills/ManualMaterialBillDetailModel.js
/*
* 
*/
Ext.define('Fusion.warehouse.model.bill.ManualMaterialBills.ManualMaterialBillDetailModel', {
    extend: 'Ext.data.Model',
    identifier: 'uuid',
    idProperty: 'Id',

    requires: [
        'Ext.data.proxy.Rest',
        'Ext.data.identifier.Uuid'
    ],

    fields: [
        { name: 'Id', type: 'string', convert: null },
        { name: 'SerialNumber', type: 'int' },
        { name: 'ManualMaterialBillNo', type: 'string' },
        { name: 'TeamCode', type: 'string' },
        { name: 'TeamName', type: 'string' },
        { name: 'MachineCode', type: 'string' },
        { name: 'MachineName', type: 'string' },
        { name: 'BrandCode', type: 'string' },
        { name: 'BrandName', type: 'string' },
        { name: 'SyncMoveBillNo', type: 'string' },
        { name: 'ManualMaterialBillDetailStatus', type: 'string' },
        { name: 'ProductionBeginTime', type: 'string' },
        { name: 'ProductionEndTime', type: 'string' },
        { name: 'OriginWorkNo', type: 'string' },
        { name: 'CreateTime', type: 'string' },
        { name: 'UpdateTime', type: 'string' },
        { name: 'RowVersion', type: 'string' }
    ],

    proxy: {
        type: 'rest',

        api: {
            read: 'Api/ManualMaterialBill/FindManualMaterialBillDetail',
            create: 'Api/ManualMaterialBill/AddManualMaterialBillDetail',
            update: 'Api/ManualMaterialBill/ModifyManualMaterialBillDetail',
            destroy: 'Api/ManualMaterialBill/RemoveManualMaterialBillDetail'
        }
    },

    getName: function () {
        return this.get('MachineCode')
    }
});

File: THOK/Fusion创建新模块所需的文件列表和文件/src/Application/Fusion.Context.Warehouse.Application/Models/Bill/ManualMaterialBills/ManualMaterialBillDetailModel.cs
using System;
using Fusion.Context.Warehouse.Domain.Models.Bill.ManualMaterialBills;

/*
* 2020-07-31 Create by wuzuqiang
*/
namespace Fusion.Context.Warehouse.Application.Models.Bill.ManualMaterialBills
{
    public class ManualMaterialBillDetailModel
    {
        public Guid Id { get; set; }
        public int SerialNumber { get; set; }
        public string ManualMaterialBillNo { get; set; }
        public string TeamCode { get; set; }
        public string TeamName { get; set; }
        public string MachineCode { get; set; }
        public string MachineName { get; set; }
        /// <summary>
        /// 班次，1:夜,2:白,3:中
        /// </summary>
        public string ShiftCode { get; private set; }
        public string ShiftName { get; private set; }
        public string BrandCode { get; set; }
        public string BrandName { get; set; }
        public string OriginWorkNo { get; set; }
        public ManualMaterialBillDetailStatus ManualMaterialBillDetailStatus { get; set; }
        public DateTime ? ProductionBeginTime { get; set; }
        public DateTime ? ProductionEndTime { get; set; }
        public DateTime CreateTime { get; set; }
        public DateTime UpdateTime { get; set; }
        public byte[] RowVersion { get; set; }

        public string ManualMaterialBillDetailStatusName { get; set; }
    }
}
